README.md

```diff
@@ -1,6 +1,5 @@
 # Boulder - An ACME CA
 
-[![Build Status](https://travis-ci.com/letsencrypt/boulder.svg?branch=main)](https://travis-ci.com/letsencrypt/boulder)
```

Comment:
Replace with
`[![Build Status](https://github.com/letsencrypt/boulder/actions/workflows/boulder-ci.yml/badge.svg?branch=main)](https://github.com/letsencrypt/boulder/actions/workflows/boulder-ci.yml?query=branch%3Amain)` to get [![Build Status](https://github.com/letsencrypt/boulder/actions/workflows/boulder-ci.yml/badge.svg?branch=main)](https://github.com/letsencrypt/boulder/actions/workflows/boulder-ci.yml?query=branch%3Amain)

Reply:
I thought we talked about this in Sprint Planning and decided we wouldn't update the badges. It even made it into the issue.

Reply:
Yep, that is the consensus we came to during Sprint planning. I also don't feel strongly about this; @aarongable if you want to keep the badge I'm cool with that.

Reply:
I'm just going to add it since at least three of us are indifferent

Reply:
Oh sorry, I'd forgotten about that! Yeah, I'm kinda neutral on 'em. They're mostly useful as a way to link to the CI, rather than as a way to show the current status, IMO. But also it's like... they're so easy to include, you might as well?